src/runtime/interop.cs

```diff
@@ -69,27 +68,66 @@ public ModulePropertyAttribute()
         }
     }
 
+    internal static partial class TypeOffset
+    {
+        static TypeOffset()
+        {
+            Type type = typeof(TypeOffset);
+            FieldInfo[] fields = type.GetFields();
+            int size = IntPtr.Size;
```

Comment:
I think it is fine to move `TypeOffset` sconstructor from generated files to `interop.cs`, but the template for the `interop*.cs` should also be updated.

Reply:
@lostmsu Ah right, good point.  Side topic - python 3.5 was EOL today, can I submit an issue to drop 3.4 and 3.5?

Reply:
Yep, that's a good idea.

Reply:
@lostmsu https://github.com/pythonnet/pythonnet/issues/1225